### ISSUE TITLE
Start with sound

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sshyran/jquery-tubular#5]
```
I found a terrific application, but could not get to start by making sound, is 
this possible? Adding 'true ' or 'false' or on / off? appreciate the response.
congratulations on the job!
```

Original issue reported on code.google.com by `djatlan...@gmail.com` on 11 Apr 2011 at 8:45

**Comment by GoogleCodeExporter** [comment]
```
I'm having the same problem. I want my video to start playing with sound, but 
can't figure out how.
```

Original comment by `matthew....@gmail.com` on 16 May 2011 at 9:27

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the comments.  I will address this soon and make it a configuration 
option.
```

Original comment by `seanmcca...@gmail.com` on 27 May 2011 at 4:00
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
In tubular.js, I took out the line refernecing mute....it still mutes when 
clicked, but plays when loaded
```

Original comment by `charleym...@gmail.com` on 2 Jun 2011 at 12:18

**Comment by GoogleCodeExporter** [comment]
```
Make changes to tubular.js on line 48:

Use this to mute on start

    // Mute at start
    ytplayer.mute()


Or this to unmute on start

        // unMute at start
    ytplayer.unMute();
    $('#videoMute').addClass('videoMute');
```

Original comment by `dreihund...@gmail.com` on 19 Jun 2011 at 6:53

**Comment by GoogleCodeExporter** [comment]
```
I guess the line order changed - can you show us where to put in that line to 
mute?  Line 48 didn't work...

It's strange by the way, that on line 56, it already says "ytplayer.mute()", 
and never says UNmute, so I'm surprised the audio would play at all...
```

Original comment by `geekfabu...@gmail.com` on 6 Jul 2012 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
It seems, that the player is playing audio all the time.

This one is already in the JS file: ytplayer.mute()

Any help will be appreciated.
```

Original comment by `yuriy.b...@gmail.com` on 10 Jul 2012 at 1:06

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I rewrote a chunk to use the new iframe API. The mute is back, the 
onYouTubePlayerReady function was not being called because it was for the old 
API. I wrote this using the new iframe API:

https://developers.google.com/youtube/iframe_api_reference

I haven't tested this thoroughly, but at least it works fine for me on firefox.
```

Original comment by `jbm...@gmail.com` on 10 Aug 2012 at 5:51

Attachments:
- [jquery.tubular.js](https://storage.googleapis.com/google-code-attachments/jquery-tubular/issue-5/comment-8/jquery.tubular.js)

**Comment by GoogleCodeExporter** [comment]
```
fixed with 1.0 release
```

Original comment by `seanmcca...@gmail.com` on 1 Oct 2012 at 10:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hey, is there any way to start the video with the sound at a certain level? 
Currently the video starts with sound at the maximum, somewhere in the middle 
would be perfect.

Thanks in advance.
```

Original comment by `paulalbe...@gmail.com` on 17 Nov 2014 at 4:40
